Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/administration-new/AdministrationGridView.tsx

```diff
@@ -37,7 +37,7 @@ interface IAdministrationGridViewProps<T> {
 }
 
 const defaultFilterToAdd = 'isdeleted~equals~false';
-const defaultSorterToAdd = 'id=DESC';
+const defaultSorterToAdd = 'id=ASC';
```

Comment:
The problems have OrderBy property. We can order them properly with it.

Reply:
Done.